src/cli/commands/private-cmds/pack-in-scope-cmd.js

```diff
@@ -0,0 +1,32 @@
+/** @flow */
+import Command from '../../command';
+import { pack } from '../../../api/scope';
+
+export default class Pack extends Command {
+  name = 'pack <id> [scopePath]';
+  description = 'pack component';
+  alias = '';
+  opts = [
+    ['d', 'directory <directory> ', 'path to store pack'],
+    ['w', 'write_bit_dependencies [boolean] ', 'write components in dependencie file'],
```

Comment:
dependencie - dependencies?
Is this specify whether to write deps to package.json? if yes write package.json here